api4/job.go

```diff
@@ -4,16 +4,21 @@
 package api4
 
 import (
+	"fmt"
 	"net/http"
+	"time"
 
 	"github.com/mattermost/mattermost-server/v5/audit"
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
+const FILE_PATH = "export/%s/%s"
```

Comment:
I don't see a need for this to be exported at a package level. Do we expect other methods to use this? If not, I would just make it a const inside the function.

Reply:
Nope! Moved inside function.